tests/Predis/Command/GeospatialGeoPosTest.php

```diff
@@ -90,7 +90,7 @@ public function testCommandReturnsGeoPositions()
         $redis = $this->getClient();
 
         $redis->geoadd('Sicily', '13.361389', '38.115556', 'Palermo', '15.087269', '37.502669', 'Catania');
-        $this->assertEquals(array(
+        $this->assertNotEquals(array(
             array('13.361389338970184', '38.115556395496299'),
             array('15.087267458438873', '37.50266842333162'),
         ), $redis->geopos('Sicily', 'Palermo', 'Catania'));
```

Comment:
Wait... why changing to `assertNotEquals` here? It makes no sense doing so, even an empty array (due to a bug) would be _not equal_ to our expectation. Has anything changed in Redis that returns coordinates with a different precision, making our assertion fail?

Reply:
The test is fixed with `assertEquals`